### PR TITLE
11298: Fix flapping SentryJob spec

### PR DESCRIPTION
**Pull request by smoyth**

### CODE REVIEW

**Comment by smoyth** [comment]
Fail is a **BRAND NEW** flapper 🤬 🤬 🤬

I logged it, shouldn't be a hard fix I think.

File: spec/jobs/sentry_job_spec.rb [hunk 1]
```diff
@@ -2,17 +2,23 @@
...
-    Raven.capture_message("Test")
-    expect(Delayed::Job.count).to eq(1)
+    VCR.use_cassette("sentry", match_requests_on: %i[method uri host path]) do
+      begin
+        1 / 0
+      rescue ZeroDivisionError => e
+        Raven.capture_exception(e)
+      end
+      Delayed::Worker.new.work_off
+    end
```
Comment (cooperka) [inline spec/jobs/sentry_job_spec.rb:22]: How does this now assert that it really creates the job? Would VCR throw if it doesn't end up handling any requests?

**Review by cooperka — APPROVED** [review]
Just a question, VCR seems great for this!

File: spec/jobs/sentry_job_spec.rb (hunk 1 above)
Comment (smoyth) [reply spec/jobs/sentry_job_spec.rb:22]: Yes. See the VCR settings in rails_helper. There is an allow_unused_requests or something.

**Pull request merged**